### DISCUSSION TITLE
[V3] Different @entangle than v2

### DISCUSSION THREAD

**Discussion by psos** [livewire/livewire#6101] [1. Help]
Hi there!

In v2 clicking wire:click span, causes change alpine value.
In v3 entangle does it only one way.

Maybe should be written in docs as breaking change  referring to $wire.get()  ?
in v3 when we should use entangle ? 

```
 namespace App\Livewire;
 
 use Livewire\Component;
 
 class Test extends Component
 {
 public $open = false;
 public function changeOpen() {
 $this->open = !$this->open;
 }
```

```
<div x-data="{
         open : @entangle('open').live,
     }">
     Open lv: <span wire:click="changeOpen">{{ $open  ? 'tak' : 'nie '}}</span><br>
     Open alpine: <span @click="open = !open" x-text="open"></span>
</div>
```

**Comment by psos** [comment]
I noticed problem with Laravel Folio and entangle

**Comment by N1ebieski** [comment]
It works for me... (but i don't use Folio)

[screen-recording(2).webm](https://github.com/livewire/livewire/assets/14810714/ec01378b-d4cd-4c53-ae04-a012dbae7537)
